src/applications/edu-benefits/tests/components/EducationWizard.unit.spec.jsx

```diff
@@ -124,4 +124,40 @@ describe('<EducationWizard>', () => {
       .true;
     expect(tree.subTree('.usa-alert-warning')).not.be.be.false;
   });
+  it('should record user events for newBenefit', () => {
+    const tree = SkinDeep.shallowRender(<EducationWizard />);
+    expect(global.window.dataLayer.length).to.equal(0);
+    answerQuestion(tree, 'newBenefit', 'yes');
+    expect(global.window.dataLayer.length).to.equal(1);
+    answerQuestion(tree, 'newBenefit', 'no');
+    expect(global.window.dataLayer.length).to.equal(2);
+  });
```

Comment:
I'm not sure that these tests are clear, or that they're testing what they say that they are testing. Did you try using a stub or a spy for `recordEvent` to make sure that it was called with the right arguments?

Reply:
Hey @bkjohnson, that is a fair point. I will add in some tests to make sure that the correct values are present vs just counting the length on each event.